tests/rptest/tests/double_bytes_sent_test.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import random
+from ducktape.mark.resource import cluster
+from ducktape.utils.util import wait_until
+
+from rptest.clients.kafka_cli_tools import KafkaCliTools
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.types import TopicSpec
+
+
+class BytesSentTest(RedpandaTest):
+    """
+    Produce and consume some data then confirm the bytes sent metric.
+    """
+    topics = (TopicSpec(), )
+
+    def __init__(self, test_context):
+        super(BytesSentTest, self).__init__(test_context=test_context)
+
+        self.kafka_tools = KafkaCliTools(self.redpanda)
+
+    def _bytes_sent(self):
+        bytes_sent = 0
+
+        for node in self.redpanda.nodes:
+            # Convert the metrics generator to a list
+            metrics = list(self.redpanda.metrics(node))
+
+            # Find the metric family that tracks bytes sent from kafka subsystem
+            family_filter = filter(
+                lambda fam: fam.name == "vectorized_kafka_rpc_sent_bytes",
+                metrics)
+            family = next(family_filter)
+
+            # Sum bytes sent
+            for sample in family.samples:
+                bytes_sent += sample.value
+
+        return bytes_sent
+
+    @cluster(num_nodes=3)
+    def test_bytes_sent(self):
+        num_records = 10240
+        records_size = 512
+        MB = 1048576
+
+        # Produce some data (10240 records * 512 bytes = 5MB of data)
+        self.kafka_tools.produce(self.topic,
+                                 num_records,
+                                 records_size,
+                                 acks=-1)
+
+        # Establish the current counter which won't be zero, but
+        # we don't really care what the actual value is
+        start_bytes = self._bytes_sent()
+        self.logger.debug(f"Start bytes: {start_bytes}")
+
+        # Consume all the records
+        results = self.kafka_tools.consume(self.topic, num_records).split('\n')
```

Comment:
I'm not sure we need a new specialized consumer for this -- something like RpkConsumer could do the job, then get the byte count either by just summing the length of the messages, or lightly modify RpkConsumer to have a byte-counting mode to avoid stashing messages we won't look at.